EI patterns/aggregator/README.md

```diff
@@ -0,0 +1,36 @@
+## Integration with Aggregator Pattern Using Ballerina Integrator (BI)
+
+### Overview
+
+The **Aggregator** patiently collects a sequence of messages and combines them once all have been received.
+This integration is built using **Ballerina Integrator (BI)** to showcase the ease of implementing such patterns within a low-code integration environment.
+
+For more detailed information on the **Aggregator** pattern, visit the [Aggregator documentation](https://www.enterpriseintegrationpatterns.com/patterns/messaging/Aggregator.html).
+
+## Design View
+
+The **Design View** visualizes the overall system structure.
+
+![Design View](design.png)
+
+## Integration Flow
+
+![Flow Diagram](flow.png)
+
+## Sequence Diagram
+
+![Flow Diagram](sequence.png)
+
+## Steps to Open with Ballerina Integrator (BI)
+
+Follow these steps to open the project and start working with the **Aggregator** integration using **Ballerina Integrator (BI)** in **VSCode**:
+
+1. **Clone the Repository:**
```

Comment:
IMO no need to bold only this step. 

```
1. Clone the repository to your local machine by running the following command.
   ```bash
   git clone https://github.com/wso2/docs-bi.git
   ```
2. Open VSCode.
3. Once VSCode is opened, go to File > Open Folder....
4. Navigate to the directory where you cloned the repository.
5. Select the project folder and open it.
```

Reply:
Fixed